src/modules/Dropdown/Dropdown.js

```diff
@@ -755,6 +756,38 @@ export default class Dropdown extends Component {
     return _.findIndex(options, ['value', value])
   }
 
+  getDropdownAriaOptions = (ElementType) => {
+    const { loading, disabled, search, multiple } = this.props
+    const { open } = this.state
+    const ariaOptions = {
+      role: 'listbox',
+      'aria-busy': loading,
+      'aria-disabled': disabled,
+      'aria-expanded': !!open,
+    }
+    if (search) {
+      ariaOptions.role = 'combobox'
+    } else if (ElementType === Menu.Item) {
+      ariaOptions.role = 'menuitem'
+    } else {
+      ariaOptions['aria-multiselectable'] = multiple
+    }
+    return ariaOptions
+  }
+
+  getDropdownMenuAriaOptions() {
+    const { search, multiple } = this.props
+    const ariaOptions = {}
+
+    if (search) {
+      ariaOptions['aria-multiselectable'] = multiple
+      ariaOptions.role = 'listbox'
+    } else if (getElementType(Dropdown, this.props) === Menu.Item) {
+      ariaOptions.role = 'menu'
```

Comment:
Hm, I see the necessity for checking if the Dropdown is a MenuItem now since this is a child and there is no other way to get this role added with the current components.

I'll have to think about this one.  This component is doing so much.  We've considered ways to break it up and also make the trigger/menu separate components.  We also have a need to add an explicit prop for the "as menu item" and "as button" use cases.  Either of these paths would make this aria prop addition cleaner as well.